extensions/cornerstone/src/utils/setCornerstoneLayout.js

```diff
@@ -0,0 +1,22 @@
+import { redux } from '@ohif/core';
+
+const { setLayout } = redux.actions;
+
+/**
+ * Update the current layout with a simple Cornerstone one
+ *
+ * @return void
+ */
+const setCornerstoneLayout = () => {
+  const layout = {
+    numRows: 1,
+    numColumns: 1,
+    viewports: [{ plugin: 'cornerstone' }],
+  };
+
+  const action = setLayout(layout);
+
+  window.store.dispatch(action);
```

Comment:
I think we're stuck doing this until we make it possible to control things like layout through a service. The only other option would be to make the store available to extensions.

I'm just hesitant to codify that.